assembl/tests/views_tests/test_graphql_vote_session.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+
+from assembl.graphql.schema import Schema as schema
+
+
+def snake_to_camel(string):
+    # https://stackoverflow.com/questions/19053707/converting-snake-case-to-lower-camel-case-lowercamelcase
+    components = string.split('_')
+    # We capitalize the first letter of each component except the first one
+    # with the 'title' method and join them together.
+    return components[0] + "".join(x.title() for x in components[1:])
+
+
+def assert_langstring_is_equal(langstring_name, fetched_model, source_model):
+    source_value = getattr(source_model, langstring_name).entries[0].value
+    fetched_value = fetched_model[snake_to_camel(langstring_name) + 'Entries'][0]['value']
+    assert source_value == fetched_value
+
+
+def assert_langstrings_are_equal(langstrings_names, fetched_model, source_model):
+    for langstring_name in langstrings_names:
+        assert_langstring_is_equal(langstring_name, fetched_model, source_model)
+
+
+fragments = u"""
+    fragment voteSessionGlobals on VoteSession {
+      headerImage {
+        title
+        mimeType
+        externalUrl
+      }
+    }
+
+    fragment langStringEntry on LangStringEntry {
+      localeCode
+      value
+    }
+
+    fragment voteSessionLangstringsEntries on VoteSession {
+      titleEntries {
+        ...langStringEntry
+      }
+      subTitleEntries {
+        ...langStringEntry
+      }
+      instructionsSectionTitleEntries {
+        ...langStringEntry
+      }
+      instructionsSectionContentEntries {
+        ...langStringEntry
+      }
+      propositionsSectionTitleEntries {
+        ...langStringEntry
+      }
+    }
+"""
+
+
+def en_value(entry):
+    return entry[0]['value']
```

Comment:
lolz, maybe only true for test case. Should probably check the locale for en. Again, it's testing, so the code will pass, but this is semantically not correct.